arch/risc-v/src/qemu-rv/qemu_rv_irq.c

```diff
@@ -108,17 +108,17 @@ void up_disable_irq(int irq)
   int extirq;
   uint32_t oldstat;
 
-  if (irq == QEMU_RV_IRQ_MTIMER)
+  if (irq == RISCV_IRQ_MTIMER)
     {
       /* Read mstatus & clear machine timer interrupt enable in mie */
 
       asm volatile("csrrc %0, mie, %1"
                   : "=r"(oldstat)
                   : "r"(MIE_MTIE));
     }
-  else if (irq > QEMU_RV_IRQ_MEXT)
+  else if (irq > RISCV_IRQ_MEXT)
```

Comment:
shouldn't we have `irq >= RISCV_IRQ_MEXT` here?

Reply:
No need for that since RISCV_IRQ_MEXT would not be enabled.

Reply:
I just see `if (RISCV_IRQ_MEXT <= irq)` in `arch/risc-v/src/qemu-rv/qemu_rv_irq_dispatch.c` so wonder why we have such difference

Reply:
Maybe we can refine this behavior in future, but it's better to keep origin code in this PR.

Reply:
Ok